test/e2e/deployment.go

```diff
@@ -183,7 +183,7 @@ func testRollingUpdateDeploymentEvents(f *Framework) {
 		"name": "sample-pod",
 		"pod":  "nginx",
 	}
-	rcName := "nginx-controller"
+	rcName := "nginx-controller-2"
```

Comment:
Was trying to fix e2e test flake; this change is not required and is now removed.